inc/blocks/ab-test/edit.css

```
@@ -0,0 +1,13 @@
+button.ab-test-xb-preview__tab {
+	background-color: #4667de;
+	padding: 0.25rem 0.5rem;
+	font-size: 0.75rem;
+	margin-right: 0.25rem;
```

Similar here for margin:
```suggestion
	margin: 0 0.25rem 0 0;
```